src/platform/site-wide/sass/shame.scss

```diff
@@ -295,8 +295,8 @@ $medium-phone-screen: 375px;
     #homepage-benefits,
     #homepage-news,
     .va-h-ruled--stars,
-    .va-nav-breadcrumbs,
-    article,
+    //.va-nav-breadcrumbs,
+    //article,
```

Comment:
commented out some selectors, but this might be a little too broad. See below where I started using params on the include. That might be a safer way to handle this?